CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.29.0-beta.0](https://github.com/MyPureCloud/genesys-spark/compare/v4.28.1...v4.29.0-beta.0) (2024-03-25)
```

Comment:
What's the reasoning in having a changelog generated for this PR but not other PRs? Should a changelog only be generated for deployments?